src/side-navigation/internal.tsx

```diff
@@ -117,10 +115,11 @@ export function ItemList({ variant, items, activeHref, fireChange, fireFollow }:
 
 interface DividerProps {
   variant: 'default' | 'header';
+  role?: 'presentation' | 'separator';
 }
 
-function Divider({ variant = 'default' }: DividerProps) {
-  return <hr className={clsx(styles.divider, styles[`divider-${variant}`])} />;
+function Divider({ variant = 'default', role = 'separator' }: DividerProps) {
+  return <hr className={clsx(styles.divider, styles[`divider-${variant}`])} role={role} />;
```

Comment:
`hr` has implicit aria role separator. I guess we could leave it undefined if it is not presentation.